client/me/purchases/manage-purchase/index.jsx

```diff
@@ -239,12 +253,43 @@ class ManagePurchase extends Component {
 		}
 
 		return (
-			<CompactCard tagName="button" displayAsLink onClick={ this.handleRenew }>
-				{ translate( 'Renew Now' ) }
+			<CompactCard tagName="button" displayAsLink onClick={ onClick }>
+				{ content }
 			</CompactCard>
 		);
 	}
 
+	renderRenewNowNavItem() {
+		const { translate } = this.props;
+		return this.renderRenewalNavItem( translate( 'Renew Now' ), this.handleRenew );
+	}
+
+	renderRenewAnnuallyNavItem() {
+		const { translate, purchase, relatedMonthlyPlanPrice } = this.props;
+		const annualPrice = getRenewalPrice( purchase ) / 12;
+		const savings = Math.round(
+			( 100 * ( relatedMonthlyPlanPrice - annualPrice ) ) / relatedMonthlyPlanPrice
+		);
+		return this.renderRenewalNavItem(
+			<div>
+				{ translate( 'Renew Annually' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **29** times:
`translate( 'renews annually', { context: 'Informative text for renewals in /checkout'} )` *ES Score: 11*